### PR TITLE
fix: required flutter version

### PR DESCRIPTION
**Pull request by ianmaciel**
Bucause very_good_cli depends on Flutter >= 3.7.1 the build fails on Github Actions created by this same CLI.

This commit update the Actions script included on project creation in order to use a more recent Flutter version.

fix: #656

This PR are blocked by the PR listed below. This PR **must** be updated after merging these PR:
- https://github.com/VeryGoodOpenSource/very_good_core/pull/243 (or [#242](https://github.com/VeryGoodOpenSource/very_good_core/pull/242))
- https://github.com/VeryGoodOpenSource/very_good_flutter_package/pull/44

## Description

Update the templated included on

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ X ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/src/commands/create/templates/very_good_core/very_good_core_bundle.dart [hunk 1]
```diff
@@ -1626,15 +1626,15 @@ final veryGoodCoreBundle = MasonBundle.fromJson(<String, dynamic>{
...
       "type": "text"
     }
   ],
   "name": "very_good_core",
   "description": "A Very Good Flutter app created by Very Good Ventures.",
-  "version": "0.1.13",
+  "version": "0.1.14",
   "environment": {"mason": ">=0.1.0-dev.40 <0.1.0"},
-  "repository": "https://github.com/VeryGoodOpenSource/very_good_core",
+  "repository": "https://github.com/ianmaciel/very_good_core",
```
Comment (renancaraujo) [inline lib/src/commands/create/templates/very_good_core/very_good_core_bundle.dart:1637]: Extraneous change

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# [v0.11.0+2]
+
+- fix: upgrade bricks to `flutter 3.7.3` to meet very_good_analysis requirements (requires at least `flutter 3.7.1`)
```
Comment (renancaraujo) [inline CHANGELOG.md:3]: Extraneous change, we update changelog on release prs

File: pubspec.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 name: very_good_cli
 description: A Very Good Command-Line Interface for Dart created by Very Good Ventures.
-version: 0.11.0+1
+version: 0.11.0+2
```
Comment (renancaraujo) [inline pubspec.yaml:3]: Extraneous change

File: lib/src/commands/create/templates/very_good_core/very_good_core_bundle.dart (hunk 1 above)
Comment (ianmaciel) [reply lib/src/commands/create/templates/very_good_core/very_good_core_bundle.dart:1637]: I'm sorry, this was'nt supposed to be commited.

**Pull request closed without merging**